test/layouts/test_columns.py

```diff
@@ -62,11 +62,11 @@ def test_columns_window_focus_cycle(qtile):
     qtile.c.window.toggle_floating()
     qtile.testWindow("three")
 
-    # test preconditions, columns adds clients at pos of current, in two stacks
-    assert qtile.c.layout.info()['columns'][0]['clients'] == ['one']  # 'three', 'two', 'one']
+    # test preconditions, columns adds clients at pos after current, in two stacks
+    assert qtile.c.layout.info()['columns'][0]['clients'] == ['one']
     assert qtile.c.layout.info()['columns'][1]['clients'] == ['three', 'two']
     # last added window has focus
     assertFocused(qtile, "three")
 
     # assert window focus cycle, according to order in layout
-    assertFocusPath(qtile, 'two', 'float1', 'float2', 'one', 'three')
+    assertFocusPath(qtile, 'float1', 'float2', 'one', 'three', 'two')
```

Comment:
Looks like this is failing: https://travis-ci.org/qtile/qtile/jobs/454683349#L901